_research/ai-models/flow-patterns-complex-flows/2026-viscoelastic-jets.md

```
@@ -0,0 +1,134 @@
+1. **Dimensionality reduction through modal decompositions**
+We demonstrate that the complex dynamics can be interpreted as a superposition of a few coherent structures.
+
+2. **Reduced-order modeling with hybrid machine learning**
+We describe a surrogate model that combines modal decompositions with a deep neural network for temporal forecasting. 
+
+---
+
+## Coherent structures
+We first show the application of modal decompositions for identifying the dominant coherent structures in the viscoelastic turbulent planar jet. 
+
+Modal decompositions decompose the data into a superposition of modes, each accompanied by characteristic values that represent either their energy content or dynamical traits. Modal decompositions require large datasets, and they can be used, for example, for [reconstructing flow fields from sparse measurements](https://modelflows.github.io/modelflowsapp/research/2026-from-sensors-to-3d-reconstruction/). Here, we use modal decompositions for compressing high-dimensional data into a more interpretable low-dimensional form by means of the higher order dynamic mode decomposition (HODMD); more information about the method and applications is available [here](https://modelflows.github.io/modelflowsapp/software/notebooks/2026-modaldecomposition/).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)